src/ops/cli/helmfile.py

```diff
@@ -38,6 +40,8 @@ def get_epilog(self):
             ops data/env=dev/region=va6/project=ee/cluster=experiments/composition=helmfiles helmfile sync
             # Run helmfile sync for a single chart
             ops data/env=dev/region=va6/project=ee/cluster=experiments/composition=helmfiles helmfile sync -- --selector chart=nginx-controller
+            # Run helmfile sync with concurrency flag
+            ops data/env=dev/region=va6/project=ee/cluster=experiments/composition=helmfiles helmfile --concurrency=1 sync -- --selector chart=nginx-controller
```

Comment:
Switch sync and --concurrency order here to keep it in line with what gets executed by helmfile

Reply:
In fact, it won't work running with an additional option followed by extra arguments like `--concurrency=1 -- --selector chart=test`